geomesa-index-api/src/test/scala/org/locationtech/geomesa/index/conf/partition/TablePartitionTest.scala

```diff
@@ -74,13 +73,19 @@ class TablePartitionTest extends Specification {
     }
 
     "extract partitions from disjoint filters" in {
-      val filter = ECQL.toFilter("dtg < '2018-01-01T00:00:00.000Z' AND dtg > '2018-02-01T00:00:00.000Z'")
-      val partitionOption = TablePartition(ds, sft)
-      partitionOption must beSome
-      val partition = partitionOption.get
-      val partitionsOption = partition.partitions(filter)
-      partitionsOption must beSome
-      partitionsOption.get must beEmpty
+      val filters = Seq(
+        "dtg < '2018-01-01T00:00:00.000Z' AND dtg > '2018-02-01T00:00:00.000Z'",
+        "dtg >= '2014-01-01T12:00:00.000Z' and dtg < '2014-01-01T12:00:00.001Z' and dtg < '2014-01-01T12:00:00.000Z'",
```

Comment:
Is `and dtg < '2014-01-01T12:00:00.000Z` repeated on purpose?

Reply:
the first one is `.001Z` !